seqr/views/apis/variant_search_api.py

```diff
@@ -58,23 +60,9 @@ def query_variants_handler(request, search_hash):
     variants, total_results = get_es_variants(results_model, sort=sort, page=page, num_results=per_page,
                                               skip_genotype_filter=is_all_project_search, user=request.user)
 
-    response_context = {}
-    if is_all_project_search and len(variants) == total_results:
```

Comment:
this was an optimization put in place so that when we searched across every family in seqr we only returned context for the matched families, as loading the context for every family took 2-3 minutes. Since this is now the default behavior for all searches, and that context now takes ~6 seconds to load, this is not needed